go/vm/geth/ct.go

```diff
@@ -163,9 +163,26 @@ func convertGethStackToCtStack(state *geth_vm.GethState, stack *st.Stack) *st.St
 }
 
 type callInterceptor struct {
-	context vm.RunContext
-	stateDb *stateDbAdapter
-	static  bool
+	parameters vm.Parameters
+	stateDb    *stateDbAdapter
+	static     bool
+}
+
+func (i *callInterceptor) makeCall(kind vm.CallKind, value vm.Value, sender, recipient vm.Address, data []byte, gas uint64) ([]byte, uint64, error) {
```

Comment:
nit: the parameters `sender`, `recipient`, `value`, `data`, and `gas` are passed to create a `vm.CallParameter` struct. I think it would be nicer to pass them already as a strut, since it would give callers a sort of named-parameter syntax, documenting the parameter use.